src/app/pages/BooksPage/slice/saga.ts

```diff
@@ -0,0 +1,54 @@
+import { BOOK_ENDPOINTS } from 'app/configs/endpoints';
+import { call, put, takeLatest } from 'redux-saga/effects';
+import { request } from 'utils/request';
+import { BooksActions as actions, BooksActions } from '.';
+
+export function* BookSaga(action) {
+  try {
+    yield call(request, BOOK_ENDPOINTS.ListBook, {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
```

Comment:
There should not be a body in a `GET` request that is supposed to return the list of books and not add books.
This also does not take into account that a list of books is returned as a result of calling the `ListBook` API endpoint. (See comment about state/store)